client/homescreen/activity-panel/index.js

```diff
@@ -42,32 +48,48 @@ export const ActivityPanel = () => {
 	const panels = getAllPanels( panelsData );
 
 	return (
-		<Accordion>
-			<Fragment>
-				{ panels.map( ( panelData ) => {
-					const {
-						className,
-						count,
-						id,
-						initialOpen,
-						panel,
-						title,
-						collapsible,
-					} = panelData;
-					return (
-						<AccordionPanel
-							key={ id }
-							className={ className }
-							count={ count }
-							initialOpen={ initialOpen }
-							title={ title }
-							collapsible={ collapsible }
-						>
-							{ panel }
-						</AccordionPanel>
-					);
-				} ) }
-			</Fragment>
-		</Accordion>
+		<Panel className="woocommerce-activity-panel">
+			{ panels.map( ( panelData ) => {
+				const {
+					className,
+					count,
+					id,
+					initialOpen,
+					panel,
+					title,
+					collapsible,
+				} = panelData;
+				return collapsible ? (
+					<PanelBody
+						title={ [
+							<Text key={ title } variant="title.small">
+								{ title }
+							</Text>,
+							count !== null && <Badge count={ count } />,
+						] }
+						key={ id }
+						className={ className }
+						initialOpen={ initialOpen }
+						collapsible={ collapsible }
+						disabled={ ! collapsible }
+					>
+						<PanelRow>{ panel }</PanelRow>
+					</PanelBody>
+				) : (
+					<div className="components-panel__body">
+						<h2 className="components-panel__body-title">
+							<Button
+								className="components-panel__body-toggle"
+								aria-expanded={ false }
+								disabled={ true }
```

Comment:
The disabled styling is causing it to be faded, could we change it, so it isn't faded, just like below?
<img width="525" alt="Screen Shot 2021-01-04 at 3 04 31 PM" src="https://user-images.githubusercontent.com/2240960/103569712-29eb0700-4e9e-11eb-8a5d-f20c8795f179.png">

Also when opening a panel below this placeholder isn't showing a gap
<img width="539" alt="Screen Shot 2021-01-04 at 3 09 15 PM" src="https://user-images.githubusercontent.com/2240960/103570382-62d7ab80-4e9f-11eb-8055-993240f2ab85.png">

Reply:
>The disabled styling is causing it to be faded, could we change it, so it isn't faded, just like below?

I think this is expected behavior.  Disabled buttons should provide visual cues that they are in fact disabled.  This is the default GB Button component behavior.

>Also when opening a panel below this placeholder isn't showing a gap

I did my best to follow the previous designs, but I may have been mistaken or the component was already broken.  I agree that this gap looks odd, but seemed intentional based on the existing css.  /cc @octaedro @elizaan36 @pmcpinto for any hints or reasoning on this gap.

Reply:
I agree that the difference between the gaps looks a bit odd. @elizaan36 what do you think?

Reply:
Yeah, the gap is intentional to indicate which panel is open so there should be a gap between the open panel and closed panels. Originally we were only going to have one open at a time, but currently since they can all be open then there would be the consistent gap between all panels.

Reply:
>but currently since they can all be open then there would be the consistent gap between all panels.

Does this mean the gap should exist even when they're closed?  Or still only on open panels?

Reply:
The gap indicates that a panel is open, and appears between adjacent panels when one accordion is expanded. When the accordion is closed and the adjacent panels are also closed (or empty), there's no gap.